stories/dem-info.stories.jsx

```diff
@@ -0,0 +1,15 @@
+import DemInfo from '../app/components/dem-info';
+import React from 'react';
+
+export default {
+    component: DemInfo,
+    args: {}
+}
+
+export const Primary = { args: {
+    user: {
+        dob: new Date('1990-10-20'),
```

Comment:
Please make dob a Date.toString().  The reason being that when objects are passed through socket.io to the browser JSON.stringify() is used and so binary objects end up as strings.  So I've learned it's easier to write code that could run on the server or the browser if you just define all the database properties to be strings in the first place.

Reply:
Gotcha, appreciate the insight, made the change!